sdk/dataprotection/Azure.ResourceManager.DataProtection/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2022-10-10)
+
+### Breaking Changes
+
+New design of track 2 initial commit.
+
+### Package Name
+
+The package name has been changed from `Microsoft.Azure.Management.DataProtection.Backup` to `Azure.ResourceManager.DataProtection`.
```

Comment:
Compared to other languages, it looks like the track 1 package name has an extra suffix `Backup`. For instance, this RP in python is named [azure-mgmt-dataprotection](https://pypi.org/project/azure-mgmt-dataprotection/1.0.0b1/).

Have reached to the service team to double confirm the package name.

Reply:
What's the name for Java?

Reply:
It is [azure-resourcemanager-dataprotection](https://search.maven.org/artifact/com.azure.resourcemanager/azure-resourcemanager-dataprotection/1.0.0-beta.1/jar).

Reply:
Confirmed with service team, backup is a correct suffix, so use `Azure.ResourceManager.DataProtectionBackup` as the namespace to keep consistent with track 1.